app/utils/auth.ts

```diff
@@ -38,6 +38,15 @@ export const getHasuraRoleName = (roles?: string[]): string => {
   }
 };
 
+/**
+ * Check if a user has any of the provided role names
+ * @param {string[]} roles - an array of roles to check for
+ * @param {CustomUser} user - the user object
```

Comment:
i read that for typescript u can leave out type declarations in the docstrings bc it can be repetitive since they are also listed in the param declarations below but nbd

Reply:
removed 👍